Ports/.port_include.sh

```diff
@@ -25,19 +25,21 @@ target_env
 DESTDIR="${SERENITY_INSTALL_ROOT}"
 
 enable_ccache() {
-    if command -v ccache &>/dev/null; then
-        ccache_tooldir="${SERENITY_BUILD_DIR}/ccache"
-        mkdir -p "$ccache_tooldir"
-        if [ "$SERENITY_TOOLCHAIN" = "Clang" ]; then
-            for tool in clang clang++; do
-                ln -sf "$(command -v ccache)" "${ccache_tooldir}/$tool"
-            done
-        else
-            for tool in gcc g++ c++; do
-                ln -sf "$(command -v ccache)" "${ccache_tooldir}/${SERENITY_ARCH}-pc-serenity-${tool}"
-            done
+    if [ "${USE_CCACHE:-true}" = "true" ]; then
+        if command -v ccache &>/dev/null; then
```

Comment:
can we combine these to avoid indenting everything by another level?

Reply:
yep, looks like.

el test for the new impl:
```
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ export USE_CCACHE=false
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ if [ ${USE_CCACHE:-true} = "true" ] && command -v ccachc &>/dev/null; then echo "hi"; fi
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ if [ ${USE_CCACHE:-true} = "true" ] && command -v ccache &>/dev/null; then echo "hi"; fi
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ unset USE_CCACHE
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ if [ ${USE_CCACHE:-true} = "true" ] && command -v ccache &>/dev/null; then echo "hi"; fi
hi
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ if [ ${USE_CCACHE:-true} = "true" ] && command -v ccachc &>/dev/null; then echo "hi"; fi
andrew@Andrew-Workstation:~/serenity/Ports/OpenJDK$ 
```